src/components/ExpenseForm.js

```diff
@@ -2,101 +2,158 @@ import { useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import styled from 'styled-components';
 
-export default function ExpenseForm({ addToExpenses }) {
+export default function ExpenseForm({ members, addToExpenses }) {
   const [inputValue, setInputValue] = useState({
     title: '',
     description: '',
     amount: '',
     date: new Date().toISOString().split('T')[0],
+    paidby: '',
+    paidfor: [],
   });
   const navigate = useNavigate();
   const handleSubmit = e => {
     e.preventDefault();
-
+    console.log(inputValue);
     addToExpenses(prevExpenses => [
       ...prevExpenses,
       {
         title: inputValue.title,
         amount: inputValue.amount,
         description: inputValue.description,
         date: inputValue.date,
+        paidby: inputValue.paidby,
+        paidfor: inputValue.paidfor,
```

Comment:
Since all properties of inputValue are named the same as your target object you can just write:
```suggestion
       ...inputValue
```

Reply:
Awesome, thanks!